src/channel.rs

```diff
@@ -420,7 +424,7 @@ impl ChannelBuilder {
         self.build_args()
     }
 
-    /// Build an insure connection to the address.
+    /// Build a unsecure [`Channel`] that connects to a specific address.
```

Comment:
It should be insecure.

Reply:
Build **an** insecure [`Channel`] that connects to a specific address.